src/components/Pages/Content/InputForm.js

```diff
@@ -0,0 +1,101 @@
+
+import React, { useState, useCallback, useEffect } from 'react';
+
+import CourseGoalList from '../../ShowList/CourseGoals/CourseGoalList/CourseGoalList';
```

Comment:
`Course` and `Goal` has nothing to do with our project, please replace them with more appropriate name. This applicable to all other places.